crates/telio-wg/src/wg.rs

```diff
@@ -115,6 +121,81 @@ pub struct Io {
     pub libtelio_wide_event_publisher: Option<mc_chan::Tx<Box<LibtelioEvent>>>,
 }
 
+/// Keepalive packets size
+pub const KEEPALIVE_PACKET_SIZE: u64 = 32;
+/// Default wireguard keepalive duration
+pub const WG_KEEPALIVE: Duration = Duration::from_secs(10);
+
+#[derive(Copy, Clone)]
+/// docs
+pub struct BytesAndTimestamps {
+    /// docs
+    rx_bytes: u64,
+    /// docs
+    tx_bytes: u64,
+    /// docs
+    rx_ts: Instant,
+    /// docs
+    tx_ts: Instant,
+}
+
+impl BytesAndTimestamps {
+    /// docs
+    pub fn new(rx_bytes: Option<u64>, tx_bytes: Option<u64>) -> Self {
+        let now = Instant::now();
+        BytesAndTimestamps {
+            rx_bytes: rx_bytes.unwrap_or_default(),
+            tx_bytes: tx_bytes.unwrap_or_default(),
+            rx_ts: now,
```

Comment:
IMHO, we have a slightly dangerous "condition" here. As `BytesAndTimestamps` is being inserted when new node is added -> no rx or no tx has happened, ever. In current variant this is indicated using `rx_bytes` or `tx_bytes` of zero indicates that. In addition, one can only know that `rx_ts/tx_ts` is "valid" only after looking at `rx_bytes` and `tx_bytes`.

But maybe it would be clearer/easier to understand if we would use `Option<u64>` and `Option<Instant>` and simply set `None` under such conditions.

@gytsto WDYT?

Reply:
Yup, you are totally right. Will do that

Reply:
updated